frontend/src/utils.ts

```diff
@@ -221,6 +221,29 @@ export function getTestedPerDayChartRows(
   return [["Dátum", "Pozitívne", "Negatívne"], ...data];
 }
 
+export function getTestedPerDayPercentChartRows(
+  logs: InfectedIncreaseLog[],
+  timespan: number
+) {
+  const data = logs
+    .filter((item) => {
+      if (timespan < 0) {
+        return true;
+      }
+      const date = new Date(item.date);
+      const from = subDays(new Date(), timespan);
+      return date > from;
+    })
+    .map((item) => {
+      const date = new Date(item.date);
+      return [
+        format(date, "yyyy-MM-dd"),
+        item.infected_increase / item.tests_increase,
```

Comment:
What if `item.tests_increase` is 0 ? I am not sure but that is probably breaking this chart when all records are visible

Reply:
Added ternary operator. If `item.tests_increase` is 0, the resulting value will be 0.